admin/plugins.php

```diff
@@ -55,6 +55,54 @@ function perflab_get_standalone_plugins() {
 	);
 }
 
+/**
+ * Returns an array mapping of standalone plugins to existing wpp modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return string[]
+ */
+function perflab_get_standalone_plugins_module_map() {
+	return array(
+		'webp-uploads'          => 'images/webp-uploads',
+		'dominant-color-images' => 'images/dominant-color-images',
+	);
+}
+
+/**
+ * Returns an array mapping of standalone plugins main file to existing wpp modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return string[]
+ */
+function perflab_get_standalone_plugins_file_map() {
+	return array(
+		'webp-uploads'          => 'webp-uploads/load.php',
+		'dominant-color-images' => 'dominant-color-images/load.php',
+	);
+}
+
+/**
+ * Returns an array of standalone plugins with currently active modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return string[]
+ */
+function perflab_get_standalone_plugins_with_active_modules() {
+	$modules                                = perflab_get_module_settings();
+	$standalone_plugins_with_active_modules = array_filter(
+		perflab_get_standalone_plugins_module_map(),
+		static function ( $v ) use ( $modules ) {
+			return ! empty( $modules[ $v ] ) && $modules[ $v ]['enabled'];
+		},
+		ARRAY_FILTER_USE_BOTH
+	);
+
+	return $standalone_plugins_with_active_modules;
```

Comment:
See above. Also why do we need `ARRAY_FILTER_USE_BOTH`?

I would suggest something like this:
```suggestion
	$modules = perflab_get_module_settings();
	return array_filter(
		array_keys( perflab_get_standalone_plugins_constants() ),
		static function ( $v ) use ( $modules ) {
			return ! empty( $modules[ $v ] ) && $modules[ $v ]['enabled'];
		}
	);
```

It should be sufficient to simply return the module slugs here. Maybe we can then rename the function to `perflab_get_active_modules_with_standalone_plugins()` to clarify?